release-team/release-team-onboarding.md

```diff
@@ -0,0 +1,65 @@
+
+The Kubernetes Community defines Members as the following:
+
+> Members are continuously active contributors in the community. They can have issues and PRs assigned to them, participate in SIGs through GitHub teams, and pre-submit tests are automatically run for their PRs. Members are expected to remain active contributors to the community.
+>
+> Defined by: Member of the Kubernetes GitHub organization
+
+The requirements for becoming a member are enumerated in the [k/community repo documentation](https://github.com/kubernetes/community/blob/master/community-membership.md#member). Generally, you'll need a few public contributions and some existing community members to sponsor you. The Release Team is a great place to meet members of the community that might be able to meet sponsors!
+
+For new shadows, it's not always necessary to become a member of the GitHub organization right away. Talk to the lead for the Release Team role you're shadowing for the definitive answer. For some roles (Enhancements for example) it is important for Shadows to be members of the GitHub organization because they have to maintain milestones in the [k/enhancements](https://github.com/kubernetes/enhancements) repo. For other roles (Release Notes for example), it's less urgent for new Shadows to be members of the GitHub organization. Your mileage may vary, but you should try to become an official member of the Kubernetes Community as soon as possible.
+
+## All Members
+
+### Contact Sheet
+
+Each release has a Release Team contact sheet which is a source of truth for who has which role in the Release Team and what their contact information is. The Release Lead usually makes this very early on in the release. Make sure that you have filled in your contact information in the contact sheet as soon as you become a member of the Release Team.
+
+### Mailing Lists
+
+There are a few mailing lists that you should be aware of. Depending on your relationship with the Release Team, you may want to join one or more of the following lists:
+
+#### Release Team Mailing List
+
+All members of the current release team should be apart of this list. If you're on the Release Team, please join this list: https://groups.google.com/forum/#!forum/kubernetes-release-team
+
+#### Milestone Burn Down Mailing List
+
+All members of the release team and anyone who attends Release Team meetings / keeps up with the release process should join the Milestone Burndown group: https://groups.google.com/forum/#!forum/kubernetes-milestone-burndown
+
+## Role Leads
+
+The leads of each role need some access. If this is your first time as a lead, please go through the following list and make sure you have the required access.
+
+### Milestone Maintainers
+
+The leads of all roles should be able to manage issue milestones (often in [k/enhancements](https://github.com/kubernetes/enhancements)) using the Prow `/milestone` command. To gain access as a Milestone Maintainer, please talk to the Release Lead.
```

Comment:
we should probably link to some form of documentation about this command. 
maybe https://prow.k8s.io/command-help#milestone and https://github.com/kubernetes/test-infra/tree/master/prow